literature.bib

```diff
@@ -2019,6 +2028,23 @@ @inproceedings{KNF+:SPLC17
 	year = 2017
 }
 
+@inproceedings{YCP:SPLC17,
+author = {Young, Bobbi and Cheatwood, Judd and Peterson, Todd and Flores, Rick and Clements, Paul},
+title = {Product Line Engineering Meets Model Based Engineering in the Defense and Automotive Industries},
+year = 2017,
+isbn = {9781450352215},
+publisher = ACM,
+address = NY,
+url = {https://doi.org/10.1145/3106195.3106220},
+doi = {10.1145/3106195.3106220},
+booktitle = SPLC,
+pages = {175–179},
+numpages = {5},
+keywords = {feature-based product line engineering, feature models, PLE factory, model-based engineering, product configurator, feature profiles, Product line engineering, variation points},
```

Comment:
Are these "official" keywords or tags by you? For the latter, please use sth. like cs-tags = {...}

Reply:
These are official keywords.

Reply:
lol